content/blog/2016/2016-12-31-what-a-year.adoc

```diff
@@ -0,0 +1,193 @@
+---
+layout: post
+title: Thank you for an amazing 2016
+tags:
+- jam
+- jenkins2
+- pipeline
+- blueocean
+- azure
+- gsoc
+author: rtyler
+---
+
+
+image:/images/jenkins-happy-new-year.png["Happy New Year from Jenkins", role=right]
+
+I do not think it is an exaggeration to say: 2016 was the best year yet for the
+Jenkins project. Since the first commit in 2006, the project has reached a
+number of significant milestones in its ten years but we have never experienced
+the breadth of major milestones in such a short amount of time. From
+link:/2.0/[Jenkins 2]
+and
+link:/projects/blueocean[Blue Ocean]
+to the
+link:/projects/gsoc[Google Summer of Code]
```

Comment:
I've noticed I've forgotten to send a PR with this page update :( Will do it soon